src/ralph/backends/lrs/async_mongo.py

```diff
@@ -0,0 +1,57 @@
+"""Async MongoDB LRS backend for Ralph."""
+
+
+import logging
+from typing import Iterator, List
+
+from ralph.backends.data.async_mongo import AsyncMongoDataBackend
+from ralph.backends.lrs.base import (
+    BaseAsyncLRSBackend,
+    StatementParameters,
+    StatementQueryResult,
+)
+from ralph.backends.lrs.mongo import MongoLRSBackend
+from ralph.exceptions import BackendException, BackendParameterException
+
+logger = logging.getLogger(__name__)
+
+
+class AsyncMongoLRSBackend(BaseAsyncLRSBackend, AsyncMongoDataBackend):
+    """Async MongoDB LRS backend implementation."""
+
+    settings_class = AsyncMongoDataBackend.settings_class
```

Comment:
Do we need to redefine settings_class, as this backend inherits from `AsyncMongoDataBackend`?

Reply:
Indeed we don't!

Reply:
Since `AsyncMongoLRSBackend` is derived from both `BaseAsyncLRSBackend` and `AsyncMongoDataBackend`, each defining attribute `settings_class`, we have to redefine `settings_class` as it would have the `BaseAsyncLRSBackend` value (resolution order from left to right)